### DISCUSSION TITLE
Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.

### DISCUSSION THREAD

**Discussion by jihea-park** [vercel/next.js#65975] [Help]
### Summary

I made an app by next.js with the `npx create-next-app@latest my-app --typescript --tailwind --eslint`
And I did `npm run dev`.
When I saved `tailwind.config.ts`, I got the warning. (Even though I didn't change anything.)
``` 
(node:84088) Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.
```

I already added `"type": "module"` on my package.json.
But the warning still occurs.

Is there any way to solve the warning?

**Comment by jspm2013** [comment]
same here - just can upvote it...please help, we know its not related to nextjs, but the usual SO answers/fixes dont work with nextjs, thats obviously because of the things nextjs shortens for us compared to simple node/react apps...

**Comment by Arsh1219** [comment]
same issue. did anyone get solution?

**Comment by Zerebokep** [comment]
Same here.

**Comment by syi0808** [comment]
What version is your node?

**Comment by Zerebokep** [comment]
I'm on 22.3.0

**Comment by kiriny** [comment]
I encountered the same issue while running Node.js v22.4.0. However, after switching back to Node.js v20.15.0 (LTS), the warning disappeared.

**Reply by syi0808** [reply]
Me too. When I tried it node v21 safe too.

**Comment by danny900714** [comment]
It seems like is is related to the latest version of NodeJS.

**Comment by khanhquocnguyen** [comment]
I have the same issue.
Node v22.4.0

**Comment by ShayokhShorfuddin** [comment]
Same issue here ☠️
Node v22.3.0

**Comment by LeaReXx** [comment]
Same issue🤦‍♂️
Node v22.5.1
NextJS v14.1.0

**Comment by syi0808** [comment]
Can someone please provide a reproduced repository?

I'm sure I've experienced it too, but I can't reproduce it..

**Reply by josephbeuysmum** [reply]
ive encountered it by simply doing the official tutorial: https://nextjs.org/learn/dashboard-app

**Comment by shutt** [comment]
Still having this issue
Node v22.6.0
NextJS 14.2.5

**Comment by SHAGGYER** [comment]
having this issue while trying to run a custom server (Node v20.15)
